multiuser/machine-auth/che-multiuser-machine-authentication/src/main/java/org/eclipse/che/multiuser/machine/authentication/server/signature/spi/SignatureKeyDao.java

```diff
@@ -36,21 +36,19 @@ SignatureKeyPairImpl create(SignatureKeyPairImpl keyPair)
       throws ConflictException, ServerException;
 
   /**
-   * Removes signature key pair with given id.
+   * Removes signature key pair with given workspace id.
    *
-   * @param id signature key identifier
+   * @param workspaceId workspace key identifier to remove keypair from
```

Comment:
would `workspace identifier to remove keypair from` be better here? workspace key looks like new entity